feature-libs/checkout/base/components/checkout-payment-method/checkout-payment-form/checkout-payment-form.component.ts

```diff
@@ -201,11 +206,26 @@ export class CheckoutPaymentFormComponent implements OnInit {
     this.sameAsDeliveryAddress = !this.sameAsDeliveryAddress;
   }
 
-  getAddressCardContent(address: Address): Card {
+  createCard(address: Address): void {
+    combineLatest([
+      this.translationService.translate('addressCard.phoneNumber'),
+      this.translationService.translate('addressCard.mobileNumber'),
+    ]).pipe(
+      tap(([textPhone, textMobile]) => {
+        if (address) {
+          this.getAddressCardContent(address, textPhone, textMobile);
+        }
+      }
```

Comment:
I think this should be in `subscribe` over `tap`, seeing that the returned value isn't used.